app/views/spree/admin/overview/single_enterprise_dashboard.html.haml

```diff
@@ -37,13 +37,10 @@
   .alpha.seven.columns.dashboard_item.single-ent#map
     .header
       %h3
-        %span.icon-map-marker
+        %span.icon-user
         = t "your_profil_live"
-      %p
-        = t "on_ofn_map"
```

Comment:
If we're deleting this line, the key should be removed from `config/locales/en/yml` at the same time, as it's not used anywhere else in the app

Reply:
Good catch, done